docs/hugo/content/introduction/metrics.md

```diff
@@ -0,0 +1,45 @@
+# ASOv2 Metrics
+
+Prometheus metrics are exposed for ASOv2 which can be helpful in diagnosability and tracing.
+The metrics exposed fall into two groups: Azure based metrics and reconciler metrics (from `controller-runtime`).
+
+## Disabling the metrics
+
+By default, metrics scraping is turned on for ASOv2. It can be turned off using the Helm chart 
+`--set metrics.enable` or by omitting `--metrics-addr` flag from the Deployment yaml.
+
+### ASOv2 Helm Chart
+
+While installing the Helm chart, we can turn the metrics ON and OFF and set the metrics expose address using the 
```

Comment:
Suggest using markdown for emphasis instead of capitals.

```suggestion
While installing the Helm chart, we can turn the metrics _**on**_ and _**off**_ and set the metrics expose address using the 
```